velox/dwio/dwrf/writer/Writer.cpp

```diff
@@ -526,6 +526,9 @@ void Writer::close() {
 }
 
 void Writer::abort() {
+  // NOTE: we need to reset column writer as all its dependent objects (e.g.
```

Comment:
Would it make sense to extract this bug fix into a separate PR?

Reply:
This is required for this PR, otherwise abort will run into memory free exception (the virtual method exception)